docs/topics/backend-details.rst

```diff
@@ -256,6 +256,8 @@ your development platform.
 +----------------------+--------------------+----------------------------+--------------------------------------+
 | REDIS_URL            | Zamboni            | URL to redis               | redis://localhost:6379               |
 +----------------------+--------------------+----------------------------+--------------------------------------+
+| SIGNING_SERVER       | Zamboni            | URL to signing server      |                                      |
```

Comment:
the default is an empty string which means it falls back to having no signing server